### ISSUE TITLE
Rendimiento de CPU != Rendimiento del ordenador

### ISSUE THREAD

**Issue by emilioj** [myTeachingURJC/Arq-computadores-01#3]
Hola!

Cuando habláis del rendimiento de un ordenador, mezcláis la idea de CPU=ordenador: «El rendimiento de un ordenador X, será mayor que el de un ordenador Y, si tarda menos tiempo en ejecutar un programa. Es decir, si su tiempo de CPU es menor (por tanto es más rápido)»

Yo pondría «... si su tiempo de ejecución es menor», y ya si queréis, especificais que en ese tiempo de ejecución hay varios tiempos incluídos. Esto lo podéis enlazar en algún punto de la wiki con una descripción de lo que es un sistema cpu-bound, memory-bound o I/O bound, que creo es un ONTOPIC interesante.

**Comment by Obijuan** [comment]
Sí, es cierto. En esta parte introductoria se ha hecho así para simplificar. A los estudiantes que se les da esta asignatura no han visto sistemas operativos y no quería entrar en excesivos detalles en este punto para no complicarles la vida. El objetivo en estas primeras sesiones es que capten la idea. No obstante, tienes toda la razón, así que pondré notas a pie de página (o en otro lugar) para aclarar todos esos aspectos.

La manera en la que suelo dar las asignaturas es mediante "refinamientos sucesivos". Les cuento medias-verdades, para simplificar, para que vayan entendiendo las ideas y puedan practicar desde el primer momento. Luego más adelante, les refino las idea. "¿Os acordáis de que os dije esto? Pues no es del todo verdad porque...."  y se refina y concreta más la idea

Muchas gracias por el feedback
